src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Perf.Comments.cs

```diff
@@ -41,18 +39,14 @@ public partial class Utf8JsonReaderCommentsTests
         public void Setup()
         {
             _jsonPayload = Encoding.UTF8.GetBytes(s_testCases[TestCase]);
-
-            if (MultiSegment)
-            {
-                _jsonPayloadSequence = GetSequence(_jsonPayload, SegmentSize);
-            }
+            _jsonPayloadSequence = GetSequence(_jsonPayload, 100);
```

Comment:
you've mentioned 4096 would be a better test

Reply:
We should re-consider the test, sure, but I am primarily making a cosmetic change here.